src/main/kotlin/com/sourcegraph/cody/autocomplete/CodyAutocompleteManager.kt

```diff
@@ -124,7 +119,12 @@ class CodyAutocompleteManager {
    * @param editor The editor instance to provide autocomplete for.
    * @param offset The character offset in the editor to trigger auto-complete at.
    */
-  fun triggerAutocomplete(editor: Editor, offset: Int, triggerKind: InlineCompletionTriggerKind) {
+  fun triggerAutocomplete(
+      editor: Editor,
+      offset: Int,
+      triggerKind: InlineCompletionTriggerKind,
+      lookupString: String?
```

Comment:
After taking a look at the usages, I have a small suggestion: `lookupString: String? = null`.